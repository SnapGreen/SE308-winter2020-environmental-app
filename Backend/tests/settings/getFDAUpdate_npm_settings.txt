settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	CURRENTLATEST: FoodData_Central_branded_food_csv_2020-04-29.zip,202005010424,74439754
	LASTLATEST: FoodData_Central_branded_food_csv_2020-04-29.zip,202005010424,74439754
	SERVER_POPULATED: false
	FDC_DIR_ADDRESS: https://fdc.nal.usda.gov/fdc-datasets
	TMPDIR: Backend/temp/
	TMPFILE_END: .tmp
	TMPLINKSFILE: Backend/temp/links.tmp
	TMPFILELIST: Backend/temp/available_data.tmp
	USAGE:
		Usage: ./getFDAUpdate.sh [OPTION] (use option -h for help)
	HELP:
		Usage: ./getFDAUpdate.sh [OPTION] (use option -h for help)
		**If no OPTION supplied, debug mode on (temp files remain)
			-b: bypass debug mode, download only if new
			-f: force download
			-n: test relative to repo root
			-ns: output settings relative to repo root
			-s: output settings only
			-t: test mode (silent)
			-h: print help
