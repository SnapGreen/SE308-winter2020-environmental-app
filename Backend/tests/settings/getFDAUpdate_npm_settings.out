		Usage: ./getFDAUpdate.sh [OPTION] (use option -h for help)
